.eslintrc

```diff
@@ -26,6 +26,10 @@
     "no-restricted-syntax": "off",
     // disable the rule for all files
     "@typescript-eslint/explicit-module-boundary-types": "off",
+    "react/function-component-definition": [
+      2,
```

Comment:
what does this 2 stand for?

Reply:
I think it is for "error" [see ESLint docs](https://eslint.org/docs/latest/use/configure/rules#rule-severities), but I will change it to "error" to be more specific.